src/Effects.js

```diff
@@ -1541,6 +1541,40 @@ module.exports = class Effects {
       }
     };
 
+    this.growing_stars = {
+      stars: [],
+      colorIndex: 0,
+      starSpacing: 30,
+      numStars: 9,
+
+      init: function () {
+        p5.angleMode(p5.RADIANS);
+        this.colorIndex = 0;
+        this.stars = [];
+        for (var i = 0; i < this.numStars; i++) {
```

Comment:
draw loop was something like: 
''for(var i = 0; i < this.numStars; i++){
 p5.fill(colorFromPalette(colorIndex + i));
...
}" but I'm not sure if that buys us any additional efficiency. Not ship-blocking, but something the consider.

Reply:
Sorry not sure I follow?

Reply:
Gah - sorry! I'm not sure where the first part of that comment went, but I edited it to include the first part of the sentence. Let me know if it still doesn't make sense and I'll elaborate!

Reply:
Ah ok makes sense. I don't think that works in this case because the index in the array changes as the stars grow (when a star gets too big it's pulled off the front of the array and a new star is added to the back of the array), so it ends up looking kind of choppy:
![Nov-08-2019 11-42-08](https://user-images.githubusercontent.com/8787187/68505768-e60a5c80-021c-11ea-856a-f7960ad6ce69.gif)

Reply:
Got it - thanks for investigating!